tests/test_configuration.py

```diff
@@ -14,164 +14,150 @@
 """
 Unit tests for the :mod:`pennylane` configuration classe :class:`Configuration`.
 """
-# pylint: disable=protected-access
-import unittest
+import pytest
 import os
 import logging as log
 
 import toml
 
-from defaults import pennylane, BaseTest
+from defaults import pennylane
 import pennylane as qml
 from pennylane import Configuration
 
 log.getLogger('defaults')
 
+@pytest.fixture(scope="session")
+def config_path():
+    return 'default_config.toml'
```

Comment:
Since this is simply constant test data, probably best to keep it as a module variable rather than convert it to a fixture.

Reply:
Will do!